source/agora/flash/Channel.d

```diff
@@ -148,7 +148,7 @@ public class Channel
         ManagedDatabase db)
     {
         this.flash_conf = flash_conf;
-        this.conf = conf;
+        this.conf = cast()conf;
```

Comment:
Hum, `in` means `scope`, so not a good idea. Also `cast`s are evil, why do we need this ?

Reply:
without the cast;

```
source/agora/flash/Channel.d(151,21): Error: cannot implicitly convert expression `conf` of type `const(ChannelConfig)` to `ChannelConfig
```

It can't be copied for some reason I guess.

Reply:
Yeah, let's leave it out of the `in` changes then

Reply:
Done